cpp/src/IceSSL/SSLEngine.cpp

```diff
@@ -96,7 +102,7 @@ IceSSL::SSLEngine::verifyPeerCertName(const string& address, const ConnectionInf
 
         bool certNameOK = false;
         string addrLower = IceUtilInternal::toLower(address);
-        bool isIpAddress = IceInternal::isIpAddress(address);
+        bool isIpAddress = IceInternal::isIpAddress(string(address));
```

Comment:
Is this extra string necessary?

Reply:
No this is a leftover from the first version that used a `string_view`